core/controllers/acl_decorators.py

```diff
@@ -2753,6 +2753,73 @@ def test_can_accept_suggestion(
     return generate_decorator_for_handler
 
 
+def can_update_suggestions(handler):
+    """Function that generates a decorator for a given handler.
```

Comment:
This comment doesn't say much. Can you make sure it notes that it checks whether the user can update suggestions? The below decorator is a good example.

Reply:
Done